clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -3266,10 +3254,8 @@ public void testBeginningOffsetsTimeout(GroupProtocol groupProtocol) {
         );
     }
 
-    // TODO: this test triggers a bug with the CONSUMER group protocol implementation.
-    //       The bug will be investigated and fixed so this test can use both group protocols.
     @ParameterizedTest
-    @EnumSource(value = GroupProtocol.class, names = "CLASSIC")
+    @EnumSource(value = GroupProtocol.class, names = {"CLASSIC", "CONSUMER"})
```

Comment:
It seems removing `names` is more simple, right?

Reply:
You're right. It's simpler. I updated it.